src/Appwrite/Platform/Workers/Usage.php

```diff
@@ -4,22 +4,22 @@
 
 use Exception;
 use Utopia\CLI\Console;
-use Utopia\Database\Database;
-use Utopia\Database\DateTime;
 use Utopia\Database\Document;
 use Utopia\Platform\Action;
+use Appwrite\Event\UsageDump;
 use Utopia\Queue\Message;
 
```

Comment:
Add a todo that we still need to handle the case of deployments performed when usage us being aggregated.

Reply:
Add another Todo to refactor the names of the two usage workers